k8s/metrics/grafana/deployment.yaml

```diff
@@ -38,23 +38,23 @@ spec:
             - name: grafana-dashboard-provider
               mountPath: /etc/grafana/provisioning/dashboards
             - name: grafana-dashboards-vald-cluster-overview
-              mountPath: /var/lib/grafana/dashboards-vald/00
```

Comment:
Grafana 11.6 complained `no such file or directory` without `dashboards` directory. So I moved the physical directory on Grafana pod. It does not change `Vald` directory on Grafana dashboard.